### DISCUSSION TITLE
Can we update to Monterey safely ?

### DISCUSSION THREAD

**Discussion by simon-caignart** [daliansky/XiaoMi-Pro#612] [General]
Thanks for the feedback!

**Comment by stevezz3** [comment]
It would be good if you use OpenCore. Intel Bluetooth might not work, and you can wait for the next release to fix it.

**Comment by dldygnl** [comment]
is the bluetooth issue fixed in the newest EFI v 1.6.3 ?

**Reply by stevezz3** [reply]
Yes

**Comment by AungPaingHtoo** [comment]
After updating to Mac OS 12, I got boot loop in apple logo.
I think it is because I use sata drive instead of m.2.
I think i have to delete my drive and downgrade to os11. :(

**Reply by stevezz3** [reply]
You might need [CtlnaAHCIPort.kext](https://github.com/dortania/OpenCore-Install-Guide/blob/master/extra-files/CtlnaAHCIPort.kext.zip) for sata disks. Also, reset nvram might help. Using OpenCore is recommended.

**Reply by AungPaingHtoo** [reply]
For some reason, my mi notebook pro with core i7 8gen cpu cannot use opencore. I just delete the EFI of clover and put EFI of opencore and reboot.
Then after mi logo appear , it just stuck there . So i just use clover .
I will try again with the latest clover . Where should i put the CtlnaAHCIPort.kext ? Should I put it in EFI>clover>kexts>others ?

**Reply by stevezz3** [reply]
You need to add OpenCore as a boot entry.

> Where should i put the CtlnaAHCIPort.kext ? Should I put it in EFI>clover>kexts>others ?

Yes.

**Reply by AungPaingHtoo** [reply]
Yes I click F2 choose open core as main boot option or F12 to manually choose opencore to boot but my computer just could not boot using open core. I remember I use open core last year and after some update, it just didn't boot so I go back to clover.

Plus I tried to install macos12 again after updating from Catarina to Big Sur .
And I found that in Big Sur , macOS 12 didn't allow me install update to my Sata drive.
But if I update macos12 from Catarina , it allows me to update but it boot loop at apple logo.
So I think the main problem is mac12 didn't support sata currently .
May be I am too noob to figure it out lol.
Thanks for all the help and explanations.

**Reply by AungPaingHtoo** [reply]
I fresh install Catarina and put the CtlnaAHCIPort.kext in clover , I can somehow able to boot into macOS12 and it is doing just fine.
I don't understand what is the real reason why I cannot update macOS12 previously. Because previously , I tried the same steps as I did now and did not work .

Anyway all's well that ends well I guess lol.
